deployment/Dockerfile.ppc64le.centos

```diff
@@ -0,0 +1,12 @@
+FROM  ppc64le/centos:7
```

Comment:
Is it necessary to provide two image?

Reply:
you mean an Ubuntu and Centos image? Probably not.